src/main/java/com/hotels/hcommon/ssh/SshSettingsMapper.java

```diff
@@ -0,0 +1,129 @@
+/**
+ * Copyright (C) 2018 Expedia Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hotels.hcommon.ssh;
+
+import java.util.Iterator;
+import java.util.List;
+
+import javax.validation.constraints.Max;
+import javax.validation.constraints.Min;
+import javax.validation.constraints.Pattern;
+
+import org.hibernate.validator.constraints.NotBlank;
+
+import com.hotels.hcommon.ssh.SshSettings.Builder;
+import com.hotels.hcommon.ssh.validation.constraint.TunnelRoute;
+
+public class SshSettingsMapper {
```

Comment:
See comment in other PR about not wanting to bring the concept of a "metastore" into what's currently a generic ssh library. It would be nice to get rid of this class though, and we need to review what's going on with the builders and whether we really need them.